index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Meditation App</title>
+</head>
+<body>
+  <div class="meditation-app">
+    <div class="video-container">
+      <video Loop>
+        <source src="./video/rain.mp4">
+      </video>
+    </div>
+    <div class="container__time">
+      <button class="time-select" data-time="300">05 Minutes</button>
+      <button class="time-select" data-time="600">10 Minutes</button>
+      <button class="time-select" data-time="900">15 Minutes</button>
+    </div>
+    <div class="container__player">
+      <audio class="song">
+        <source src="sounds/rain.mp3">
+      </audio>
+      <img class="play" src="svg/play.svg" alt="play">
+      <svg class="track-outline" width="100%" height="100%" viewBox="0 0 453 453" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <circle cx="226.5" cy="226.5" r="216.5" stroke="white" stroke-width="20"/>
+      </svg>
+      <svg class="moving-outline" width="100%" height="100%" viewBox="0 0 453 453" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <circle cx="226.5" cy="226.5" r="216.5" stroke="#018EBA" stroke-width="20"/>
+      </svg>
+      <h3 class="time-display">00:00</h3>
```

Comment:
From a semantic point of view, using an h3 without an h2 or h1 is not good practice. I assume you have used h3 because of the most appropriate size of this heading for your needs. But you can define sizes in css and not break semantic meaning of the headers in your file.

Reply:
DONE!